src/main/java/uk/ac/ebi/ena/webin/cli/WebinCli.java

```diff
@@ -241,7 +254,11 @@ private static WebinCliParameters initParameters(String submissionAccount, Webin
                     .setPassword(parameters.getPassword())
                     .setTest(parameters.isTest())
                     .build();
-            submitService.doSubmission(bundle.getXMLFileList(), bundle.getCenterName(), getVersionForSubmission(), bundle.getManifestMd5(), getManifestFileContent());
+
+            String enaSubmissionTool = parameters.getEnaSubmissionTool() != null && !parameters.getEnaSubmissionTool().isEmpty() ?

```

Comment:
- We should capture the Webin-CLI version for all WebinSubmissionTool (EnaSubmissionTool) enum values.
- Suggestion is to remove:
```
String enaSubmissionTool = parameters.getEnaSubmissionTool() != null && !parameters.getEnaSubmissionTool().isEmpty() ?  parameters.getEnaSubmissionTool() : getVersionForSubmission();
```
and have:
```
submitService.doSubmission(bundle.getXMLFileList(), bundle.getCenterName(), getVersionForSubmission(parameters.getEnaSubmissionTool()), bundle.getManifestMd5(), getManifestFileContent());
```

where getVersionForSubmission is something like:
```
    public static String
    getVersionForSubmission(WebinSubmissionTool submissionTool) {
       String version = getVersion();
       switch (submissionTool) {
        case WEBIN-CLI-REST:
           return String.format("%s:%s",  /* TODO */, null == version ? "" : version);
        default:
           // Default to current behaviour
         return String.format("%s:%s", WebinCli.class.getSimpleName(), null == version ? "" : version);
    }
```
- Please add a unit test if possible test for getVersionForSubmission for both WEBIN-CLI and WEBIN-CLI-REST enum values